rustfst/src/algorithms/tr_unique.rs

```diff
@@ -18,6 +18,12 @@ pub(crate) fn tr_compare<W: Semiring>(tr_1: &Tr<W>, tr_2: &Tr<W>) -> Ordering {
     if tr_1.olabel > tr_2.olabel {
         return Ordering::Greater;
     }
+    //if tr_1.weight < tr_2.weight {
+    //    return Ordering::Less;
```

Comment:
commented code. is it intended ?

Reply:
Yes, this fix will be applied in another PR